build-release-tools/lib/manifest-web-ui.json

```diff
@@ -0,0 +1,17 @@
+{
+    "build-name": "rackhd-devel", 
+    "build-requirements": "xenial", 
+    "downstream-jobs": [], 
+    "repositories": [
+        {
+            "branch": "", 
+            "commit-id": "",
+            "repository": "https://github.com/RackHD/on-wss.git"
```

Comment:
wait for a second... I'm fine with using the manifest-on-web-ui.json
just to well consider 

for core services in RackHD, we build and test them together , because finally  they will be released together a whole rackhd core engine  and they can't be build/test alone. so we need manifest to lock down on-xxx commits .

go to ur on-web-ui and on-wss, 
(1) are those 2 repos highly tight with each other in code level ? 
(2) by which way are you going to release on-web-ui ( either deb or docker ) ? I don't think they will include each other or packaged together..

But now I'm thinking about SMI, they have several repos as micro services.
but I believe they will have to be tested together with RackHD & FIT.

what will be included in their manifest ? all of the SMI repos ?  rackhd core-services repo ?